ucfcs/GrowthPlus
# Mack/congrats screen

Closes issue #144. A congrats screen has been implemented that appears when the user completes the 4th game with a passing score. 

To test:
- make a user
- complete the last lesson in roadmap 4
- complete the game in roadmap 4
- if you pass, you should go to the congrats screen where it shows the avatar you chose with confetti raining down :)
- tapping the next button on the congrats screen takes you back to the landing page
- Double check that passing other games or quizzes don't take you to the congrats screen, but that has yet to happen for me which is good

The screen may not be perfect yet; I'm all ears to improvements and am willing to work on it a little more
